pkgs/tools/misc/ckb-next/default.nix

```diff
@@ -1,14 +1,14 @@
-{ stdenv, fetchFromGitHub, substituteAll, libudev, pkgconfig, qtbase, qmake, zlib, kmod }:
+{ stdenv, fetchFromGitHub, substituteAll, libudev, pkgconfig, qtbase, cmake, zlib, kmod }:
 
 stdenv.mkDerivation rec {
-  version = "0.2.9";
+  version = "0.3.0";
```

Comment:
`0.3.1` was released meanwhile, if it isn't too much, update and verify it still works (assuming you have the hardware.)